### PR TITLE
Make Specific skill page responsive for mobile, other dimensions #1952

### LINKED ISSUE #1952: Make Specific skill page responsive for mobile, other dimensions
**Actual Behaviour**

Currently, skill detail page is not responsive.

**Expected Behaviour**

Make it responsive

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1952 

Changes: Used flexbox, made skill detail page responsive

Surge Deployment Link: https://pr-1962-1-fossasia-susi-skill-cms.surge.sh

Screenshots for the change: 
Screenshot 2019-05-08 at 1 52 46 AM

### CODE REVIEW

File: src/components/SkillPage/SkillListing.js [hunk 1]
```diff
@@ -194,7 +193,7 @@ class SkillListing extends Component {
       showReportDialog,
       showAuthorSkills,
     } = this.state;
-
+    console.log(styles.home, 'Home');
```
Comment (akshatnitd) [inline src/components/SkillPage/SkillListing.js]: Remove the console.log

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Comment by harsh-jindal** [comment]
@fragm3 Yup the charts and graphs are good in mobile view.

**Comment by fragm3** [comment]
@harsh-jindal @akshatnitd made the changes, please review.

**Comment by fragm3** [comment]
@akshatnitd yes, did that on purpose as the charts need width or they would overflow

**Comment by fragm3** [comment]
@akshatnitd I had tried the ResponsiveContainer approach, there is some problem with there library. The example in ResponsiveContainer doesn't resize on window size change. Will try some alternate way.

**Comment by fragm3** [comment]
@akshatnitd I am able to resize every chart except Device Wise Skill Usage(PieChart). I have pushed the changes in my second commit. Surge Link: daffy-ocean.surge.sh, Please test the link, and compare with https://pr-1962-1-fossasia-susi-skill-cms.surge.sh.

**Comment by fragm3** [comment]
@akshatnitd @AnupKumarPanwar @hkedia321 @anshumanv please review

**Pull request merged**